gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -320,6 +369,18 @@ gis_password_page_constructed (GObject *object)
 
   validate (page);
 
+  priv->cancellable = g_cancellable_new ();
+
+  g_dbus_proxy_new_for_bus (G_BUS_TYPE_SESSION,
```

Comment:
See similar comments about it being better to build this synchronously in the g-c-c PR